contracts/Marketplace.sol

```diff
@@ -332,9 +340,16 @@ contract Marketplace is Ownable, IMarketplace2 {
         // TODO: require(purchaseAccepted, "error_rejectedBySeller")
 
         // Solidity 4:
-        // 0x91517bdd = keccak256("onPurchase(bytes32,address,uint256,uint256)")
+        // 0x4a439cc0 = keccak256("onPurchase(bytes32,address,uint256,uint256,uint256)")
         // this call returns true if beneficiary is a PurchaseListener, return value is ignored
-        p.beneficiary.call(0x91517bdd, productId, subscriber, oldSub.endTimestamp, price);
+        //(bool success, bytes memory returnData) = 
+        p.beneficiary.call(0x4a439cc0, productId, subscriber, oldSub.endTimestamp, price, fee);
+        /*
+        if(success){
+            (bool accepted) = abi.decode(returnData, (bool));
+            require(accepted, "error_rejectedBySeller");
+        }
+        */
```

Comment:
This is commented out. Should it be deleted, too? Is this feature planned?

Reply:
ability to reject a sale would enable adding white-/blacklisting into seller smart contract (e.g. Data Union). Could be a good way to remove user features from Marketplace.

Reply:
Getting this to work is the whole point of the solc5 upgrade, no?

Reply:
Ah, ok. Well, Jon mentioned in a call rejecting actually IS working. All I mean is, either that commented-out thing should be removed, or made to work. If rejecting works, that snippet probably should go.

Reply:
yes, it's working in truffle. still fixing test failures, but almost there